### ISSUE TITLE
Update langtools to work with updated bazel and remove unnecessary BUILD rules [phab:D399]

### ISSUE THREAD

**Issue by kythe** [google/kythe#629]
_Created by **shahms** at 2015-07-13 18:16:11_: 

Update langtools to work with updated bazel and remove unnecessary BUILD rules

**Comment by kythe** [comment]
_**schroederc** commented inline (context lost during migration) at 2015-07-13 18:25:58_:

This leaves us in the dangerous state of having two versions of javac (the other being `//thrid_party/openjdk:tools`).

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-07-13 18:25:58_:

I'm getting the following build error:

```
Error occurred during initialization of VM
java/lang/ClassNotFoundException: error in opening JAR file <Zip file open error> third_party/java/jdk/langtools/javac.jar
```

**Comment by kythe** [comment]
_**schroederc** required changes at 2015-07-13 18:25:58_

**Comment by kythe** [comment]
_**shahms** commented inline (context lost during migration) at 2015-07-13 18:28:15_:

I'll right, I'll look into merging these two, but it'll be a bit longer process.

**Comment by kythe** [comment]
_**shahms** wrote at 2015-07-13 18:28:15_:

>>! In (earlier comment), schroederc wrote:
> I'm getting the following build error:
> 
> ```
> Error occurred during initialization of VM
> java/lang/ClassNotFoundException: error in opening JAR file <Zip file open error> third_party/java/jdk/langtools/javac.jar
> ```

This is a problem with the Phabricator instance; it won't upload the file due to:

ERR-CONDUIT-CORE: No configured storage engine can store this file. See "Configuring File Storage" in the documentation for information on configuring storage engines.

**Comment by kythe** [comment]
_**schroederc** accepted this revision at 2015-07-13 18:32:51_

**Comment by kythe** [comment]
_**shahms** closed via commit a71101dfbfdf1105c0ab2d73bad625ea84c184f1 at 2015-07-13 19:51:44_
